queries.sql

```diff
@@ -0,0 +1,183 @@
+INSERT INTO mydb.user (username, email, password, registration_date)
+VALUES ('john_smith', 'jsmith@gmail.com', 'password', CURDATE());
+
+INSERT INTO mydb.category (name, description)
+VALUES ('Electronics', 'Electronic devices and accessories');
+
+INSERT INTO mydb.product (name, description, price, stock_qty, category_id)
+VALUES ('iPhone 15', 'Latest smartphone from Apple', 1599.99, 50, 1);
+
+INSERT INTO mydb.address (address_line1, address_line2, city, state, zip_code)
+VALUES ('123 Main St', 'Apt 4B', 'New York', 'NY', '10001');
+
+INSERT INTO mydb.useraddress (user_id, address_id)
+VALUES (4, 1);
+
+INSERT INTO mydb.order (user_id, order_date, total_amount)
+VALUES (4, CURDATE(), 1599.99);
+
+INSERT INTO mydb.orderitem (order_id, product_id, qty, price)
+VALUES (2, 1, 1, 1599.99);
+
+INSERT INTO mydb.review (user_id, product_id, rating, comment)
+VALUES (4, 1, 4.8, "The best");
+
+INSERT INTO mydb.cart (cart_id, creation_date)
+VALUES (1, CURDATE());
+
+INSERT INTO mydb.usercart (user_id, cart_id)
+VALUES (4, 1);
+
+UPDATE mydb.user
+SET password = 'newpassword'
+WHERE user_id = 4;
+
+UPDATE mydb.user
+SET password = 'StrongerPassword123.'
+WHERE user_id = 4;
+
+UPDATE mydb.category
+SET description = 'Electronic devices and accessories for everyone'
+WHERE category_id = 1;
+
+UPDATE mydb.product
+SET stock_qty = 100
+WHERE product_id = 1;
+
+UPDATE mydb.product
+SET stock_qty = 80
+WHERE product_id = 1;
+
+UPDATE mydb.address
+SET address_line2 = '62'
+WHERE address_id = 1;
+
+UPDATE mydb.address
+SET address_line1 = '72 S James St'
+WHERE address_id = 1;
+
+UPDATE mydb.review
+SET rating = 3.0
+WHERE review_id = 1;
+
+UPDATE mydb.review
+SET rating = 5.0
+WHERE review_id = 1;
+
+UPDATE mydb.review
+SET comment = "The last phone I will ever buy"
+WHERE review_id = 1;
+
+ALTER TABLE mydb.user
+ADD last_name varchar(255);
+
+ALTER TABLE mydb.user
+ADD first_name varchar(255);
+
+ALTER TABLE mydb.address
+ADD country varchar(100);
+
+ALTER TABLE mydb.product
+ADD on_sale TINYINT;
+
+ALTER TABLE mydb.order
+ADD express_shipping TINYINT;
+
+DELETE FROM mydb.orderitem WHERE order_id = 2;
+DELETE FROM mydb.order WHERE order_id = 2;
+DELETE FROM mydb.review WHERE review_id = 2;
+DELETE FROM mydb.usercart WHERE user_id = 4;
+DELETE FROM mydb.useraddress WHERE user_id = 4;
+DELETE FROM mydb.user WHERE user_id = 4;
+DELETE FROM mydb.address WHERE address_id = 1;
+DELETE FROM mydb.product WHERE product_id = 1;
+DELETE FROM mydb.category WHERE category_id = 2;
+DELETE FROM mydb.cart WHERE cart_id = 1;
+
+SELECT * FROM mydb.user AS u
```

Comment:
DO not just show everything. Try to make this query a valuable from business respective. Select some fields that will answer some business question:
Products of which categories where mostly purchased by the specific user in last month?
Did users product cart changed when changing delivery address?